js/contextMenus.js

```diff
@@ -134,6 +135,11 @@ function tabTemplateInit (frameProps) {
     })
   }
 
+  items.push(Object.assign({},
+    CommonMenu.reopenLastClosedTabItem,
+    { enabled: WindowStore.getState().get('closedFrames').size > 0 }
```

Comment:
do you mean passing it with the event to `onTabContextMenu` ? I wasn't big on including it here either, but figured passing it with the event would mean going through Tabs' props